src/routes/api/tripRoute.js

```diff
@@ -421,4 +429,35 @@ tripRouter
     operateAcceptOrReject, acceptOrRejectRequestsController
   );
 
+/**
+ * @swagger
+ *
+ * /trips/mostTraveled:
+ *    get:
```

Comment:
I would like to request to change the API route name. I recommend using `-` hyphen to improve the readability of API

<img width="787" alt="Screen Shot 2019-12-12 at 14 27 00" src="https://user-images.githubusercontent.com/56690376/70712128-c4086c00-1ceb-11ea-85e7-2adcec0a2fcf.png">

More info are [here](https://restfulapi.net/resource-naming/)

Reply:
I have changed it, you can take a look!